docs/scenes/get-started/set-up-your-environment/SelectCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { ButtonBase, mergeClasses, useTheme } from '@expo/styleguide';
+import { useEffect, useState } from 'react';
+
+import { CALLOUT, HEADLINE } from '~/ui/components/Text';
+
+type Props = {
+  imgSrc: string;
+  darkImgSrc?: string;
+  alt: string;
+  title: string;
+  description?: string;
+  isSelected: boolean;
+  onClick: () => void;
+};
+
+export function SelectCard({
+  imgSrc,
+  darkImgSrc,
+  alt,
+  title,
+  description,
+  isSelected,
+  onClick,
+}: Props) {
+  const { themeName } = useTheme();
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  useEffect(
+    function didMount() {
+      const darkMode =
+        themeName === 'dark' ||
+        (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches);
+      setIsDarkMode(darkMode);
+    },
+    [themeName]
+  );
+
+  return (
+    <ButtonBase onClick={onClick}>
+      <div className="w-[250px] shadow-xs overflow-hidden rounded-lg flex flex-col items-center border border-default hocus:scale-[102%] transition-all hocus:shadow-sm">
+        <div
```

Comment:
```suggestion
      <div className={mergeClasses(
        "w-[250px] shadow-xs overflow-hidden rounded-lg flex flex-col items-center border border-default transition-all", 
        "hocus:scale-[102%] hocus:shadow-sm"
      )}>
        <div
```

It's a good practice to split TW class name strings for the readability, we usually use `mergeClasses` helper util for that, and split the classes by scope or state.